modules/thisbook.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable */
+class thisBook {
+  constructor() {
+    if (localStorage.getItem('Added books') === null) {
+      localStorage.setItem('Added books', JSON.stringify([]));
+    }
+  }
+
+  getStoredBooks() {
+    return JSON.parse(localStorage.getItem('Added books'));
+  }
+
+  updateStoredBooks(books) {
+    localStorage.setItem('Added books', JSON.stringify(books));
+  }
+  addNewBook(bookTitle, bookAuthor) {
+    const storedBooks = this.getStoredBooks();
+    const newBook = {
+      title: bookTitle,
+      author: bookAuthor,
+    };
+    storedBooks.push(newBook);
+    this.updateStoredBooks(storedBooks);
+    this.displayBooks();
+  }
```

Comment:
- [ ] Kindly check and fix the function to add books as it is not working as expected. 👍🏿 

### Please check application behavior below ⬇️ 
![cannot add book](https://github.com/katarighe/awesome-books-es6/assets/117144028/56bdbc37-2240-49ac-ad4b-af889d960369)